### PR TITLE
Feature/improve test data

### PR DESCRIPTION
**Pull request by 01231**
Fixes AEM-4996.

# Done is when (DoD):
- [ ] I have added UI/Unit tests for my changes
- [x] Old tests and eslint rule are not broken
- [ ] I added modified component properties to the typescript typings
- [ ] Design has been reviewed with (here name of reviewer)
- [x] My changes are accordingly to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
- [ ] My changes are well documented in the README and showcased in the stories 
- [ ] My changes are shortly added to CHANGELOG.md with the issue number
- [ ] Tested with IE

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [ ] I have made/updated the ChangeLog Section in the README

### CODE REVIEW

File: src/components/10-atoms/carousel/story.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ storyAXACarousel.add('Carousel', () => {
           its customers lead as carefree a life as possible. It protects them
           and their families, their property, and their assets against risks.
         </span>
-        <span>This is a small text.</span>
+        <span>Praise and critism? Tell us what you think.</span>
```
Comment (raphaellueckl) [inline src/components/10-atoms/carousel/story.js]: 'criticism'

File: src/components/10-atoms/checkbox/demo.js [hunk 2]
```diff
@@ -12,25 +12,18 @@ storiesOf('Components|Atoms/Checkbox/Demos', module)
...
-          </p>
-        </axa-checkbox>
-      </div>
+      <axa-checkbox
+        class="hover"
+        name="checkbox"
+        checked="false"
+        required="true"
+        ><p>
+          I agree to
```
Comment (raphaellueckl) [inline src/components/10-atoms/checkbox/demo.js:21]: "I agree to the", right?

File: src/components/10-atoms/checkbox/react/DemoCheckboxLabelAsChildren.jsx [hunk 3]
```diff
@@ -11,15 +11,14 @@ const DemoCheckboxLabelAsChildren = () => {
   return (
     <AXACheckbox checked={checked} onChange={handleCheckboxChange}>
       <p>
-        click{' '}
+        I agree to{' '}
```
Comment (raphaellueckl) [inline src/components/10-atoms/checkbox/react/DemoCheckboxLabelAsChildren.jsx:14]: "I agree to the", right?

File: src/components/10-atoms/checkbox/react/story.jsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ storiesOf('Components|Atoms/Checkbox/React', module)
   })
   .addDecorator(withKnobs)
   .add('Checkbox - uncontrolled', () => {
-    const label = text('label', 'this is a label');
+    const label = text('label', 'I agree to conditions of data protection.');
```
Comment (raphaellueckl) [inline src/components/10-atoms/checkbox/react/story.jsx:17]: "I agree to the", right?

File: src/components/10-atoms/checkbox/story.js [hunk 5]
```diff
@@ -18,7 +18,7 @@ storyCheckbox.addParameters({
 
 storyCheckbox.add('Checkbox', () => {
   const refId = text('refId', `checkbox-${createRefId()}`);
-  const label = text('label', 'this is a label');
+  const label = text('label', 'I agree to conditions of data protection.');
```
Comment (raphaellueckl) [inline src/components/10-atoms/checkbox/story.js:21]: "I agree to the", right?

File: src/components/20-molecules/dropdown/react/story.jsx [hunk 6]
```diff
@@ -16,9 +16,9 @@ storiesOf('Components|Molecules/Dropdown/React', module)
   .add('Dropdown as uncontrolled React Component', () => {
     const div = document.createElement('div');
 
-    const item1 = text('First Item', 'Item A');
-    const item2 = text('Second Item', 'Item B');
-    const item3 = text('Third Item', 'Item C');
+    const item1 = text('First Item', '< CHF 1,000');
+    const item2 = text('Second Item', 'From CHF 1,000 to 10,000');
```
Comment (raphaellueckl) [inline src/components/20-molecules/dropdown/react/story.jsx:20]: "From CHF 1,000 to CHF 10,000"

File: src/components/20-molecules/dropdown/story.js [hunk 7]
```diff
@@ -46,9 +46,9 @@ storyDropdown.add('Dropdown', () => {
       ?required="${required}"
       ?native="${native}"
       items='[
-        {"name": "Item 1", "value": "Item 1" },
-        {"name": "Item 2", "value": "Item 2" },
-        {"name": "Item 3", "value": "Item 3" }
+        {"name": "< CHF 1,000", "value": "Item 1" },
+        {"name": "From CHF 1,000 to 10,0000", "value": "Item 2" },
```
Comment (raphaellueckl) [inline src/components/20-molecules/dropdown/story.js:50]: From CHF 1,000 to CHF 10,000

File: src/components/20-molecules/dropdown/ui.test.js [hunk 8]
```diff
@@ -232,7 +232,7 @@ test('should allow setting non-initial item via items property', async t => {
     );
     return visibleItem.innerText;
   });
-  await t.expect(await getVisibleItem()).eql('Item B');
+  await t.expect(await getVisibleItem()).eql('From CHF 1,000 to 10,000');
```
Comment (raphaellueckl) [inline src/components/20-molecules/dropdown/ui.test.js:235]: From CHF 1,000 to CHF 10,000

**Review by raphaellueckl — CHANGES_REQUESTED** [review]
Minors

File: src/components/10-atoms/checkbox/demo.js (hunk 2 above)
Comment (01231) [reply src/components/10-atoms/checkbox/demo.js:21]: axa.ch shows it without "the".

Screenshot 2020-01-31 at 15 21 43

File: src/components/20-molecules/dropdown/react/story.jsx (hunk 6 above)
Comment (01231) [reply src/components/20-molecules/dropdown/react/story.jsx:20]: Is the same on axa.ch.

Screenshot 2020-01-31 at 15 27 45

**Review by raphaellueckl — APPROVED** [review]

File: src/components/10-atoms/checkbox/ui.test.js [hunk 9]
```diff
@@ -127,6 +110,18 @@ test('should set refId on label and input', async t => {
     .ok();
 });
 
+fixture('Checkbox - Disabled and checked').page(
```
Comment (danyball) [inline src/components/10-atoms/checkbox/ui.test.js:113]: Why did you moved that test?

File: src/components/10-atoms/checkbox/ui.test.js [hunk 10]
```diff
@@ -137,7 +132,7 @@ test('should be clickable + change state and render parent label', async t => {
 
   await t.click($axaCheckbox);
   await t.expect($axaCheckbox.checked).ok();
-  await t.expect($axaCheckbox.count).eql(2);
+  await t.expect($axaCheckbox.count).eql(1);
```
Comment (danyball) [inline src/components/10-atoms/checkbox/ui.test.js:135]: Why did you changed that from 2 to 1?

File: src/components/30-organisms/table/story.js [hunk 11]
```diff
@@ -24,51 +24,51 @@ storiesOf('Components|Organisms/Table', module)
...
             </tr>
           </thead>
           <tbody>
             <tr>
-              <td>Body 1</td>
-              <td class="o-table--light">Longer Body 2</td>
-              <td>Super Long Body 3</td>
+              <td>Hans</td>
+              <td class="o-table--light">33</td>
+              <td>Berlin</td>
```
Comment (danyball) [inline src/components/30-organisms/table/story.js:36]: Hans aus Berlin :D stark!

**Review by danyball — CHANGES_REQUESTED** [review]

File: src/components/10-atoms/checkbox/ui.test.js (hunk 10 above)
Comment (01231) [reply src/components/10-atoms/checkbox/ui.test.js:135]: I removed one checkbox from the demo

File: src/components/10-atoms/checkbox/ui.test.js (hunk 9 above)
Comment (01231) [reply src/components/10-atoms/checkbox/ui.test.js:113]: because the test above need the path from the top

**Review by danyball — APPROVED** [review]
Thanks for your answers

**Pull request merged**